qa/pull-tester/rpc-tests.py

```diff
@@ -105,6 +105,7 @@
     'initial_reissuance_token.py',
     'feature_blocksign.py',
     'default_asset_name.py',
+    'assetdir.py',
```

Comment:
do we plan on porting some of confidential_transactions.py here, based on assetdir stuff? Might be a good idea.

Reply:
I nearly added my tests to confidential_transactions.py as it already had some issuance code in there but it already seemed to be creeping beyond the original intent. So yes, perhaps moving some of the tests in ct into asserdir would be a good idea. Perhaps renaming it to issuance_reissuance.py as well?

Reply:
Depends on what scope you intend this test to arrive at. We can always rename later of course.